lib/matrack/queries.rb

```diff
@@ -0,0 +1,78 @@
+module Matrack
+  class Queries < DataManger
+    class << self
+
+      def table_name
+        self.to_s.to_snake_case + "s"
+      end
+
+      def uniq_id
+        "#{self.to_s.to_snake_case}_id"
+      end
+
+      def execute(query)
+        objs = []
+        rows = db_conn.execute query
+        rows.each{|row| objs << self.new(row.reject!{ |k| !k.is_a? String }) }
```

Comment:
Redundant `self` detected.<br>Space missing to the left of {.<br>Space between { and | missing.